amplfi/architectures/amplfi/architectures/flows/base.py

```diff
@@ -28,8 +29,25 @@ def __init__(
         else:
             self.embedding_context = nullcontext
 
-        if embedding_weights is not None:
-            self.embedding_net.load_state_dict(torch.load(embedding_weights))
+        def load_weights(weights: Path):
```

Comment:
Great stuff. We still wan't the ability to not pass the `embedding_weights`. So a conditional like
```python
if embedding_weights is not None:
    self.embedding_net.load_state_dict(embedding_dict)
```

Also, I don't think we should be loading in the standard scaler weights here. Were you running into a problem?

Reply:
There was no problem without scaler weights. Can we ignore them, or do they need to be loaded at a later stage?

Reply:
If you're starting a flow training, they'll just be recomputed again (and should be essentially the same). 

We can ignore them here.